src/main/java/io/github/fablabsmc/fablabs/impl/fiber/tree/ConfigAttributeImpl.java

```diff
@@ -12,12 +14,12 @@
 	public ConfigAttributeImpl(FiberId identifier, SerializableType<T> type, T value) {
 		this.identifier = identifier;
 		this.type = type;
-		this.value = value;
+		this.value = Objects.requireNonNull(value);
 	}
 
 	@Override
 	public boolean setValue(T value) {
```

Comment:
Can `value` be annotated with `@Nonnull` here?

Reply:
Yes. Since we're inconsistent with nonnull annotations I didn't add one in this case.